dataset/dictionary/y-events.json5

```diff
@@ -251,6 +251,14 @@
     notes: "v3.6 期間限定イベント「盛典と慧業」の子イベント",
     tags: [ "event" ],
   },
+  {
+    en: "Hat Guy",
+    ja: "笠っち",
+    zhCN: "阿帽",
```

Comment:
@Bill-Haku @SleepyAsh0191 Can you review if this Chinese word is correct?

Reply:
yes it's correct.

Reply:
Thanks!